docs/resources/script.md

```diff
@@ -0,0 +1,33 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "coder_script Resource - terraform-provider-coder"
+subcategory: ""
+description: |-
+  Use this resource to run a script from an agent.
+---
+
+# coder_script (Resource)
+
+Use this resource to run a script from an agent.
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `agent_id` (String) The "id" property of a "coder_agent" resource to associate with.
+- `script` (String) The script to run.
+
+### Optional
+
+- `login_before_ready` (Boolean) This option defines whether or not the user can (by default) login to the workspace before it is ready. Ready means that e.g. the script is done and has exited. When enabled, users may see an incomplete workspace when logging in.
+- `run_on_start` (Boolean) This option defines whether or not the script should run when the agent starts.
+- `run_on_stop` (Boolean) This option defines whether or not the script should run when the agent stops.
+- `schedule` (String) The schedule to run the script on. This is a cron expression.
+- `timeout` (Number) Time in seconds until the agent lifecycle status is marked as timed out, this happens when the script has not completed (exited) in the given time.
```

Comment:
Can you document what you set this to if it's a long-running process? Or I guess we want people to ampersand things `&`

Reply:
I'm going to allow long-running operations that consistently log, so I'll update this.